### ISSUE TITLE
How do we encourage, force the player to spread out and make outposts and bases to push them into the big logistics puzzles

### ISSUE THREAD

**Issue by Yarden-zamir** [Yarden-zamir/Create-Prepare-to-Dye#243]

**Comment by TulkinRB** [comment]
The "trade stations landing at a random place nearby" idea we had seems to be good.
Maybe we can also make structures (abandoned factories, etc..) meaningful to the gameplay (for example mobs that spawn there) - as long as we make them common enough and visible from a distance, so players don't need to explore too much.

**Comment by TulkinRB** [comment]
Also - It's very important to make transportation a non-issue at early game so players are able to spread out without spending too much time moving between parts of their base.
I think early game elytra and rocket automation could work, but there might be a cooler solution

**Comment by Yarden-zamir** [comment]
The trade station idea works, although it doesn't sit 100% with me. I don't fully agree about the mobility part. Elytra will be craftable, but I don't think it should be specifically for the early game because if instead we make trains easier to achieave which was already the plan the player will do that

**Comment by Yarden-zamir** [comment]
So for that need to figure out how to make the trade station immovable by create and by pistons. Then figure out how to make them land properly. Then deside on a trigger. Maybe after specific trades you get a radio device that lets you call it in like a carepackage and it drops in a random radios around you

**Comment by toastonrye** [comment]
It could be interesting to use Create Trains... I'm not familiar with them but they seem very cool, and function outside of loaded chunks. I'm a player that avoids flying as much as possible, and I like the idea of infrastructure.

Making rail infrastructure fairly easy to acquire early game maybe interesting?

**Comment by Yarden-zamir** [comment]
Agreed completelty, we already made a few things easier in regards to trains and there are a few early deals that kickstart you with rails. We also increased rail placement disttance to max which makes laying trains much easier

**Comment by Yarden-zamir** [comment]
We aim to get a few more things on that front but even then we need to make sure the player has a reason to use them

**Comment by Discord-Github-Bridge** [comment]
[<img src="https://cdn.discordapp.com/avatars/127395969997209600/b2d749c0f9159013a07a929cffd19285.webp" width="15" height="15" center=true/> **yarden.zamir** on Discord says](https://discord.com/channels/890222432605057044/1184160051422830602/1193483952850210867) 
>

**Comment by Discord-Github-Bridge** [comment]
[<img src="https://cdn.discordapp.com/avatars/127395969997209600/b2d749c0f9159013a07a929cffd19285.webp" width="15" height="15" center=true/> **yarden.zamir** on Discord says](https://discord.com/channels/890222432605057044/1184160051422830602/1193483982562668595) 
> Ah forgot I need to pin it

**Comment by Yarden-zamir** [comment]
Yup, works as expected
